examples/mass_delete_old_medias.py

```diff
@@ -0,0 +1,460 @@
+        )
+    elif not html_is_custom and not plain_is_custom:
+        logger.info(
+            'Neither HTML nor plain email template were found. '
+            'Using default email templates.'
+        )
+    else:
+        logger.info(
+            'HTML and plain email template were found. '
+            'Using custom email templates.'
+        )
+
+    medias_per_speaker = {}
+    to_fallback = []
+    for media in medias:
+        if not media.get('speaker_email').strip():
+            to_fallback.append(media)
+            continue
+        speakers_emails = (media.get('speaker_email') or '').split('|')
+        for speaker_email in speakers_emails:
+            medias_per_speaker.setdefault(speaker_email.strip(), []).append(media)
+
+    to_send = {
+        speaker_email: _prepare_mail(
+            msc,
+            sender_address,
+            speaker_email,
+            speaker_medias,
+            delete_date,
+            skip_category,
+            html_template,
+            plain_template,
+            email_subject_template,
+        ) for speaker_email, speaker_medias in medias_per_speaker.items()
+    }
+
+    if apply:
+        password = getpass.getpass(f'Type the password for "{sender_address}" and press enter: ')
+        context = ssl.create_default_context()
+        sent_count = 0
+        with smtplib.SMTP_SSL(smtp_server, 465, context=context) as smtp:
+            smtp.login(sender_address, password)
+            for recipient, message in to_send.items():
+                try:
+                    smtp.sendmail(sender_address, recipient, message)
+                except smtplib.SMTPException as err:
+                    logger.error(
+                        f'Cannot send email to "{recipient}": {err}. '
+                        'Medias will be added to the fallback recipient\'s email.'
+                    )
+                    to_fallback += medias_per_speaker[recipient]
+                else:
+                    logger.debug(
+                        f'Sent "{recipient}" an email about {len(medias_per_speaker[recipient])} medias.'
+                    )
+                    sent_count += 1
+            if to_fallback:
+                fallback_message = _prepare_mail(
+                    msc,
+                    sender_address,
+                    fallback_email,
+                    to_fallback,
+                    delete_date,
+                    skip_category,
+                    html_template,
+                    plain_template,
+                    email_subject_template,
+                )
+                smtp.sendmail(sender_address, fallback_email, fallback_message)
+                sent_count += 1
+        logger.info(f'Sent {sent_count} emails.')
+    else:
+        for speaker_email, speaker_medias in medias_per_speaker.items():
+            logger.debug(
+                f'[Dry run] Would have sent "{speaker_email}" an email about {len(speaker_medias)} medias.'
+            )
```

Comment:
print email preview in dry run mode

Reply:
If you have hundreds of emails, it will make the log unreadable.

Reply:
I agree with @sdiemer . I can log them in debug mode only, to avoid polluting higher log levels... Still, debug mode will print a lot of emails. e.g. for ru:
```
INFO:__main__:Found 36539 medias added before 2020-07-01 (size: 63.5TB).
INFO:__main__:[Dry run] 1827 emails would have been sent.
```

Reply:
Ok for debug level; i do know that this will be required. 

The usecase is: i want to test my email template; i would run without --apply and in debug mode, then once i see a few emails printed out i ctrl+C. Any better idea ?

Reply:
If the usecase is to test template, then I'd suggest a subcommand / switch that does just that with dummy data. No need to query all the medias from the api just to test a template.  Especially because such a usecase might require trial and error... the get-all API will be called every time, for no reason at all.

If the usecase is to debug which email were sent to whom with what oids, then I think logging just that info is better then logging the whole mail (html + plain versions, it's a lot).

Reply:
I think there are 2 use cases:
1) test the template <-- subcommand is a perfect solution
2) verify manually which media would be deleted <-- logging a single line is ok (as long as the line contains most variables supported by the template)